ignite/contrib/handlers/lr_finder.py

```diff
@@ -0,0 +1,390 @@
+# coding: utf-8
+
+from __future__ import division
```

Comment:
We can now remove this line since no more support for 2.7

Reply:
No problem